macos/FluentUI/Button/Button.swift

```diff
@@ -156,14 +156,14 @@ open class Button: NSButton {
 
 	private var linkedPrimaryOriginalStyle: ButtonStyle?
 
-	private var mouseDown: Bool = false {
+	public var isPressed: Bool = false {
```

Comment:
So a user of a button could change the isPressed value? Is that really a valid use case? Shouldn't that just be an internal detail?

Reply:
I want the test app to be able to put the buttons in a rest/pressed/disabled state so we can easily see each color set applied to all styles of button.  Previously I achieved this by sending mouseDown and mouseUp events to the button, but since the Action overhaul this no longer works, and I need to set the isPressed var directly.  The `internal` access level is not sufficient for this, and I couldn't figure out how to expose the property in a testable subclass.  Do you know of a better way?

It's not intended to be part of the Fluent button API, but it's also not the end of the world if a user sets this value (similar to setting isHighlighted).

Reply:
Just curious, what happens if we manually set the `state` property? https://developer.apple.com/documentation/appkit/nsbutton/1528907-state

Reply:
If I try to use `state` in place of `isPressed` then the highlight-stealing effect stops working for some reason.  Probably the system is doing something complicated. The NSButton `isHighlighted` property is similarly glitchy.

Reply:
Probably worth checking the buttonType after using state. My guess is that implicitly changes its type to a type that supports on/off states? 

But I guess the question is whether the test app really needs to be able to toggle it. Can we instead count on someone changing it to actually go press the buttons for verification and avoid unnecessarily increasing the size of our production component (even if a tiny bit)? That would also ensure that our test app code actually set patterns for how we want this used by consumers. You'd be surprised how often I find our hacky test app code copied by well-meaning consumers

Reply:
It is a huge help for the test app to be able to toggle all of the buttons together to see the effects of color scheme changes on all the button variants, and capture these states in a screenshot.